crates/config/src/config.rs

```diff
@@ -1,45 +1,54 @@
 use std::path::PathBuf;
 
 use bitcoin::Network;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 
 use crate::{bridge::RelayerConfig, btcio::BtcioConfig};
 
-#[derive(Debug, Clone, Deserialize)]
-pub struct FullNodeConfig {
-    /// host:port of sequencer rpc
-    pub sequencer_rpc: String,
-}
-
-// SequencerConfig is empty for now
-#[derive(Debug, Clone, Deserialize)]
-pub struct SequencerConfig {}
-
-#[derive(Debug, Clone, Deserialize)]
-#[serde(untagged)]
-pub enum ClientMode {
-    FullNode(FullNodeConfig),
-    Sequencer(SequencerConfig),
-}
+const DEFAULT_RPC_PORT: u16 = 8542;
+const DEFAULT_P2P_PORT: u16 = 8543;
+const DEFAULT_DATADIR: &str = "strata-data";
```

Comment:
Docstrings to at least say which configs these belongs to?
From the context, the `strata-client` full node?